lib/internal/Magento/Framework/Session/SessionManager.php

```diff
@@ -15,6 +15,11 @@
  */
 class SessionManager implements SessionManagerInterface
 {
+   /**
+    * Session destroyed threshold in seconds
+    */
+    const SESSION_DESTROYED_THRESHOLD = 300;
```

Comment:
Please, remove constant and use session lifetime value from the system configuration